index.html

```diff
@@ -85,6 +85,268 @@ <h1>
+                    </div>
+                </div>
+              </article>
+            </div>
+          </section>
+
+
+          <section class="posts">
+            <div class="contaier posts_container">
+              <article class="post">
+                <div class="image">
+                  <img src="images/Dance.png" alt="">
+                </div>
+                <div class="info">
+                  <h4>Tonic</h4>
+                  <ul>
+                    <li class="canopy">CANOPY</li>
+                    <li class="dot"><img src="/images/Dot.png" alt=""></li>
+                    <li class="developer">Back End Dev</li>
+                    <li class="dot"><img src="/images/Dot.png" alt=""></li>
+                    <li class="year">2015</li>
+                  </ul>
+                  <p>A daily selection of privately<br>
+                     personalized reads; no accounts or<br>
+                     sign-ups required.
+                  </p>
+                  <a href="html" class="btn">HTML</a>
+                  <a href="css" class="btn">CSS</a>
+                  <a href="JavaScript" class="btn">JavaScript</a>
+                  <div class="btn_see">
+                    <a href="#">See Project</a>
+                  </div>
+                </div>
+              </article>
+            </div>
+          </section>
+
+          <section class="posts">
+            <div class="contaier posts_container">
+              <article class="post">
+                <div class="image">
+                  <img src="images/professional.png" alt="">
+                </div>
+                <div class="info">
+                    <div>
+                      <h4 class="title">Multi-Post<br>
+                        Stories</h4>
+                    </div>
+                    <div>
+                      <ul>
+                        <li class="canopy">CANOPY</li>
+                        <li class="dot"><img src="/images/Dot.png" alt=""></li>
+                        <li class="developer">Back End Dev</li>
+                        <li class="dot"><img src="/images/Dot.png" alt=""></li>
+                        <li class="year">2015</li>
+                      </ul>
+                    </div>
+                    <div>
+                      <p>A daily selection of privately<br>
+                        personalized reads; no accounts or<br>
+                        sign-ups required.
+                     </p>
+                    </div>
+                    <div>
+                      <a href="html" class="btn">HTML</a>
+                    <a href="css" class="btn">CSS</a>
+                    <a href="JavaScript" class="btn">JavaScript</a>
+                    </div>
+                    <div class="btn_see">
+                      <a href="#">See Project</a>
+                    </div>
+                </div>
+              </article>
+            </div>
+          </section>
+        </div>
```

Comment:
Consider making the outer wrapper a `<section>`, with each project as a `div` or preferably an `<article>`.